IdentityCore/src/validation/MSIDB2CAuthority.m

```diff
@@ -123,6 +129,11 @@ - (BOOL)supportsClientIDAsScope
 
 #pragma mark - Protected
 
++ (NSString *)authorityType
+{
+    return @"authority_b2c";
```

Comment:
should we use constants here? I saw you added global ones.

Reply:
yep, fixed.